### PR TITLE
Oms agent upgrade

### PR DESCRIPTION
**Pull request by olga-mir**
```release-note
Upgrade log analytics to version ciprod:ciprod10182019
```

### CODE REVIEW

**Comment by asalkeld** [comment]
looks great! hopefully we can get it in really soon

**Comment by asalkeld** [comment]
@olga-mir I am not fussed this time, but we normally rebase rather than merge origin/master otherwise things get messy.

**Comment by ehashman** [comment]
Need to regenerate bindata (`make generate` iirc)

File: pkg/sync/v11/data/ConfigMap/openshift-azure-logging/container-azm-ms-agentconfig.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 apiVersion: v1
 data:
+  agent-settings: |-
```
Comment (r-dilip) [inline pkg/sync/v11/data/ConfigMap/openshift-azure-logging/container-azm-ms-agentconfig.yaml]: Remove this

File: pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml [hunk 2]
```diff
@@ -2,6 +2,12 @@ apiVersion: v1
 data:
   kube.conf: |-
     # Fluentd config file for OMS Docker - cluster components (kubeAPI)
+    #fluent forward plugin
```
Comment (r-dilip) [inline pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml]: Please make sure this file matches the file [here] https://github.com/microsoft/Docker-Provider/blob/ci_feature_prod/installer/conf/kube.conf

File: pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml [hunk 3]
```diff
@@ -43,6 +49,13 @@ data:
      run_interval 60s
      log_level debug
     </source>
+     #Kubernetes health
+     <source>
+      type kubehealth
+      tag oms.api.KubeHealth.ReplicaSet
```
Comment (r-dilip) [inline pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml]: incorrect tag. please see comment above

File: pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml [hunk 4]
```diff
@@ -62,6 +75,10 @@ data:
      metrics_to_collect cpuUsageNanoCores,memoryWorkingSetBytes
      log_level info
     </filter>
+     #health model aggregation filter
+     <filter oms.api.KubeHealth**>
```
Comment (r-dilip) [inline pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml]: incorrect tag. Sync to the file mentioned above

File: pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml [hunk 5]
```diff
@@ -198,6 +215,17 @@ data:
      retry_wait 30s
      max_retry_wait 9m
      retry_mdm_post_wait_minutes 60
+     </match>
```
Comment (r-dilip) [inline pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml]: incorrect plugin. See file mentioned above

File: pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml (hunk 2 above)
Comment (asalkeld) [reply pkg/sync/v11/data/ConfigMap/openshift-azure-logging/log-analytics-cluster-agent-config.yaml]: thank you. done in the latest commit

**Comment by jim-minter** [comment]
/lgtm
/hold

This looks fine to me but I don't know all the details.  @olga-mir @asalkeld if you're happy for this to merge, do a `hold cancel`

**Pull request merged**